src/openforms/js/components/admin/forms/ReactSelect.js

```diff
@@ -82,6 +82,7 @@ const SelectWithoutFormik = ({name, options, value, className, onChange, ...prop
     classNamePrefix="admin-react-select"
     styles={styles}
     menuPlacement="auto"
+    menuPortalTarget={document.body}
```

Comment:
this is too simple as it breaks storybook, which renders in another DOM node and the `within(canvasElement)` queries break. Instead, we should probably set up a custom context that defaults to `document.body`, but which we can override in storybook to point it to the right node, similar to what we do with the modals.

Reply:
we now have this modal context for storybook from @robinmolen, so we can apply the same pattern.